Basic-Car-Maintenance-Widget/Info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>KeychainAccessGroup</key>
+	<string>$(AppIdentifierPrefix)com.mikaelacaron.Shared</string>
```

Comment:
Same question herea bout the format (as the widget extension question

Reply:
The `.` is included in `$(AppIdentifierPrefix)`.